api/rest/mantisbt_openapi.yaml

```diff
@@ -0,0 +1,1733 @@
+openapi: 3.0.0
```

Comment:
Latest OpenAPI spec version is 3.1.0.

Reply:
I will update to `3.0.3` since that version seems to have better tooling support than `3.1.0`. We can update later, as needed.